components/EventPage/TicketTable.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import Paper from '@mui/material/Paper'
+import Typography from '@mui/material/Typography'
+import Divider from '@mui/material/Divider'
+import Select, { SelectChangeEvent } from '@mui/material/Select'
+import MenuItem from '@mui/material/MenuItem'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import InfoIcon from '@mui/icons-material/Info'
+
+export default function TicketTable() {
+  const [ticketQuantity, setTicketQuantity] = useState('1')
+
+  const handleQuantity = (e: SelectChangeEvent) => {
+    setTicketQuantity(e.target.value)
+  }
+
+  return (
+    <Paper sx={{ border: '1px solid rgba(34,36,38,.15)' }}>
+      <Typography variant="h5" fontWeight="500" padding="0.8em 1em">
+        Tickets
+      </Typography>
+      <Divider />
+      <TicketPart quantity={ticketQuantity} setQuantity={handleQuantity} />
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'flex-end',
+          padding: '1.3em',
+          gap: '15px',
+        }}
+      >
+        <Typography variant="h6" fontWeight="400">
+          Grand Total: 0.00
+        </Typography>
+        <Button variant="contained">Order Now</Button>
+        <Button variant="contained" startIcon={<InfoIcon />} color="success">
+          Refund Policy
+        </Button>
+      </Box>
+    </Paper>
+  )
+}
+
+const TicketPart = ({
```

Comment:
create an interface separately.

Reply:
Okay, also extracted to a different component file with the proper interface